1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
-# Plano de estrategia
+# ESTRATEGIA
 
 ## Objetivos 
 
 ### Visión
+Nuestra página web informará a sus usuarios sobre distintos géneros musicales a través de noticias sobre actualidad musical, críticas de álbumes o de videoclips, crónica de conciertos, información relativa sobre festivales y conciertos dentro del territorio nacional y los alrededores cercanos, como pueden serlo Francia y Portugal.
+
+Queremos atraer al máximo número de usuarios únicos posibles, por eso queremos que nuestra web sea práctica y funcional, con un diseño atractivo y de uso fácil en el que las secciones que ofrezcamos estén expuestas de forma muy clara. También creemos que ofrecer contenido variado sobre distintos géneros musicales puede ser un factor importante en la obtención de esos usuarios, así como en posibles inversores, patrocinadores o publicistas.
+
+Nuestra página web apuesta por el valor añadido en el contenido, es decir, que la información que ofrezcamos tenga un carácter único en base a la experiencia, investigación y conocimientos de los periodistas que trabajan en ella. Creemos que nuestros usuarios valorarán esa obtención de criterio a través de un contenido de calidad que, al mismo tiempo, fomente la música y la diversidad de géneros. 
 
 ### Circunstancias de uso
+- ¿Quién?
+- ¿Qué?
+- ¿Cuándo?
+- ¿Dónde?
+- ¿Porqué?
+
+¿A quienes nos dirigimos? → Nuestro público objetivo serán personas aficionadas a la música y de carácter curioso que tengan entre 16 y 40 años, ya que consideramos que quienes se encuentran dentro de esta franja de edad, tienen un conocimiento básico suficiente sobre cómo navegar por Internet y de cómo utilizar una página web.
+
+¿Qué? → A estos usuarios, nosotros les ofreceremos contenido multimedia relacionado con distintos géneros musicales que puedan resultarles interesantes, tales como noticias, entrevistas, reportajes, playlist, agenda de conciertos, crónicas sobre festivales de música, recomendaciones, crítica musical, etc. 
```

Comment:
_distintos géneros musicales que puedan resultarles interesantes_, ¿quiere decir que determinaréis qué géneros musicales interesan más a vuestros usuarios y que generaréis contenido sobre esos géneros? ¿O tenéis una lista de géneros musicales ya seleccionadao sobre la que trabajaréis?